ext/pcntl/pcntl.stub.php

```diff
@@ -83,3 +83,7 @@ function pcntl_unshare(int $flags): bool {}
 #ifdef HAVE_RFORK
 function pcntl_rfork(int $flags, int $signal = 0): int{}
 #endif
+#
```

Comment:
This extra # tag broke the `gen_stub.php` script, removing it and re-running the `gen_stub.php` script will also correctly add the `ZEND_FE` in the `function_entry` struct